_posts/2015/2015-11-16-carbon-automobiles-bebop-fashion.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Carbon, Autombiles, Bebop, and Fashion
+layout: post
+tags:
+- opensource
+---
+
+When you bought one of the early commercially available computers in 1950s, it would often come with the source code of the software that ran on it, so you could modify it to fix bugs or integrate it into your environment. Academics, researchers and corporations participated in open collaboration to write software that made computers useful.
+
+In the 60s and 70s, as software became more advanced and the cost of producing it increased, vendors began using intellectual property laws to impose copying restrictions and would often withhold the source code to prevent competitors from copying it. The result was extremely expensive pieces of hardware sold by an industry that couldn’t support them with software that didn't meet the needs of their customers.
```

Comment:
Result is overstated. Some customer needs were being met and industry exploding, but dynamic changed: secrecy rather than collaboration became seen as competitive advantage, customers became more pure consumers, taking what industry gave them, and developers were disempowered. (Even that implies a probably too rosy view of the prior hardware-driven industry, but semi-accurate from developer perspective, I gather.)